app/scripts/views/message.js

```diff
@@ -629,7 +630,7 @@ define(['jquery', 'config', 'quickblox', 'underscore', 'minEmoji', 'Helpers', 't
         createAndShowNotification(msg, isHiddenChat);
       }
 
-      if ((isHiddenChat || !window.isQMAppActive) && (message.type !== 'groupchat' || msg.sender_id !== User.contact.id)) {
+      if ((isHiddenChat || !window.isQMAppActive) && (message.type !== 'groupchat' || msg.sender_id !== User.contact.id) && Settings.settingsParams.sounds_notify) {
```

Comment:
You need to create a protected property and if you want to get some `settings.get('sounds')`